include/z64.h

```diff
@@ -802,6 +802,12 @@ typedef struct
     /* 0x01 */ u8 room;
 } EntranceEntry;
 
+typedef struct
+{
+    /* 0x00 */ u8     n_points;
+    /* 0x04 */ Vec3s* point_array;
+} Path; // size = 0x8
```

Comment:
Oh I actually just added this struct myself for en_dog. Though I think it makes more sense in z64Actor, do you guys agree? Dont think anything other than actors use paths.
Also existing documentation on cloudmodding says that the path array is a segment address. Dont know if we want to use UNK_PTR for that or Vec3s*

Reply:
segment addresses are pointers per SDK. in game, the ptrs are converted from segmented to physical, so  Vec3s is acceptable here